src/components/Card/Card.js

```diff
@@ -1,6 +1,63 @@
 import './Card.css'
 import createElement from '../../lib/createElement'
 
-export default function Card(name) {
-  return createElement('section', { className: 'Card' }, name)
+export default function Card(image, name, house, ancestry, patronus) {
+  const characterName = createElement('h3', { className: 'names' }, name)
+  const profilePicture = createElement('img', {
+    className: 'profilePicture',
+    src: image,
+  })
+
+  const characterDetails = createElement('div', {
```

Comment:
`characterDetails` could be an `ul`-HTMLElement and contain only `li`-elements. Maybe try this.
```suggestion
  const characterDetails = createElement('ul', {
```